tests/robot/resources/variables.robot

```diff
@@ -3,4 +3,7 @@
 @{ENCLAVE_SUBDOMAINS}               edi  edge  airflow  flower
 ${TEST_MODEL_ID}                    demo-abc-model
 ${TEST_MODEL_1_VERSION}             1.0
-${TEST_MODEL_2_VERSION}             1.1
\ No newline at end of file
+${TEST_MODEL_2_VERSION}             1.1
+${TEST_MODEL_VERSION}               1.0
```

Comment:
Looks like `TEST_MODEL_VERSION` is obsolete.

Reply:
the same file exists in develop and is needed for tests